src/components/BookFinder/BookList/interfaces/VolumeModel.ts

```diff
@@ -0,0 +1,13 @@
+import { ImageLinkModel } from "./ImageLinkModel";
+
+export interface VolumeModel {
+  id: string;
+  volumeInfo: {
+    imageLinks: ImageLinkModel | undefined;
+    title: string;
+    description: string;
+    authors: undefined | string[];
+    averageRating: number;
+    ratingsCount: number;
+  };
```

Comment:
It would be great if you extracted volumeInfo in its separate interface and just include it here. This is in case you will need to use it in multiple places. Instead of copying 10 lines of code, just put one.